aspen/http/mapping.py

```diff
@@ -86,39 +91,3 @@ def ones(self, *names):
             if n not in lowered:
                 lowered.append(n)
         return [self[name] for name in lowered]
-
-
-class CaseInsensitiveMapping(Mapping):
```

Comment:
We're not making `Mapping` case-insensitive, nor should we. Only HTTP headers are case-insensitive, and aspen doesn't deal with those anymore, so we can drop `CaseInsensitiveMapping`, because we're not using it and we're not supposed to.